tf2onnx/tfonnx.py

```diff
@@ -99,6 +99,12 @@ def tflist_to_onnx(node_list, shape_override):
                 continue
             elif a in ignored_attr:
                 continue
+            elif a in ["key_dtype", "value_dtype", "Tin", "Tout"]:
+                attr[a] = utils.map_tf_dtype(utils.get_tf_node_attr(node, a))
+            elif a == "output_types":
+                attr[a] = [utils.map_tf_dtype(v) for v in utils.get_tf_node_attr(node, a)]
+            elif a == "output_shapes":
+                attr[a] = utils.get_tf_output_shapes_attr(node)
```

Comment:
isn't this already stored in graph._output_shapes ?

Reply:
Without the line - we get exception from onnx complaining about unknown iterable r type.

Reply:
So onnx is expecting an attribute output_shapes or output_types ? In that case you should set this in the handler for this op and not pass it 1:1 from tensorflow because all other ops don't need this.

Reply:
removed.